CMSgov/mint-app
# [EASI-2723, EASI-2870] - Resembles existing model fullstack

# EASI-2723
# EASI-2870

## Changes and Description

BE changes have been reviewed and approved.  This PR is to cover the FE and E2E functionality

- Replaced `resemblesExistingModelWhich` with `existingModelLinks`
- Added mutation to add links in addition to general characteristics changes
- Updated readonly view to read from current/existing related models to render

![Screenshot 2023-05-09 at 9 56 30 AM](https://github.com/CMSgov/mint-app/assets/95709965/cc0d8223-2673-4a84-bce0-70a6c6fd7a96)

![Screenshot 2023-05-09 at 9 57 02 AM](https://github.com/CMSgov/mint-app/assets/95709965/2c07b365-dc78-4264-acdd-fc711f08d802)

## How to test this change

- Go to page one of general characteristics and choose to answer "Which existing models does your proposed track/model most closely resemble?" with both a mixture of existing models and MINT-made models.
- Verify persistence
- Verify readonly rendering

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
